osu.Game/Screens/OnlinePlay/Matchmaking/Screens/MatchmakingSubScreen.cs

```diff
@@ -6,37 +6,40 @@
 
 namespace osu.Game.Screens.OnlinePlay.Matchmaking.Screens
 {
-    public partial class MatchmakingSubScreen : Screen
+    public abstract partial class MatchmakingSubScreen : Screen
     {
-        public MatchmakingSubScreen()
+        public abstract PlayerPanelList.PanelDisplayStyle PlayersDisplayStyle { get; }
+        public abstract Drawable? PlayersDisplayArea { get; }
+
+        protected MatchmakingSubScreen()
         {
             RelativePositionAxes = Axes.X;
         }
 
         public override void OnEntering(ScreenTransitionEvent e)
         {
             base.OnEntering(e);
-            this.MoveToX(1).MoveToX(0, 200);
+            this.FadeInFromZero(200);
```

Comment:
I've had to change these because otherwise the animation doesn't really work well with moving screens (i.e. as `PlayersDisplayArea` moves across the screen).